insights/parsers/du_usage.py

```diff
@@ -0,0 +1,58 @@
+"""
+du - Command
+============
+
+Module for the processing of output from the ``du`` command stored in
+file.  The base class ``DiskUsage`` provides all of the functionality.
+
+This example demonstrates the funcionality of ``DiskUsage`` class.
+
+Examples:
+    >>> FILE_CONTENT = '''
+    ... COMMAND> du -sh /var/lib/mongodb
+    ...
+    ... 22G     /var/lib/mongodb
+    ... '''
+    >>> du_info.get_size('/var/lib/mongodb')
```

Comment:
we need to initialize `du_info` before using it.

Reply:
Modified.